### PR TITLE
KM71-2 Add adduser command

### PR DESCRIPTION
**Pull request by ivaruf**
See comments in #106

### CODE REVIEW

File: cmd/okctl/adduser.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		validation.Field(&o.UserEmail, validation.Required, is.Email),
+	)
+}
+
+// nolint: funlen
+func buildAddUserCommand(o *okctl.Okctl) *cobra.Command {
+	opts := &AddUserOpts{}
+	cmd := &cobra.Command{
+		Use:   "adduser [env] [email]",
+		Short: "Add a user to identitypool",
```
Comment (yngvark) [inline cmd/okctl/adduser.go:48]: I don't think the word "identitypool" means anything to people, except those familiar with Cognito already.

Is it possible to come up with more understandable wording here?

Not sure if any of these are correct, but just to attempt making a suggestion:
"Add a user to the cluster"
"Give a user access to the cluster's services"

**Review by yngvark — CHANGES_REQUESTED** [review]

File: cmd/okctl/adduser.go (hunk 1 above)
Comment (ivaruf) [reply cmd/okctl/adduser.go:48]: Ok - will clarify a bit :)

**Review by yngvark — CHANGES_REQUESTED** [review]

File: cmd/okctl/adduser.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		validation.Field(&o.UserPoolID, validation.Required),
+		validation.Field(&o.UserEmail, validation.Required, is.Email),
+	)
+}
+
+// nolint: funlen
+func buildAddUserCommand(o *okctl.Okctl) *cobra.Command {
+	opts := &AddUserOpts{}
+	cmd := &cobra.Command{
+		Use:   "adduser [env] [email]",
```
[the remark this answers is not in the document]
Comment (ivaruf) [reply cmd/okctl/adduser.go:47]: Ah, yes that is right --- we have been consistenly "wrong" , figured we should make a task to fix it accross all commands.
Actually , I just added it to our board: https://trello.com/c/Uh0EAxtV/91-fix-commands-from-optional-to-required

[the remark this answers is not in the document]
Comment (ivaruf) [reply cmd/okctl/adduser.go:47]: See that we have not even been consistent in our wrongness , hehe, this should be addressed in another PR.

**Review by yngvark — APPROVED** [review]

**Pull request merged**